File: data/code_pairs/fortran/402.f90
module constants
    implicit none

    logical, parameter :: BIN_OUTPUT = .false.
    integer, parameter :: FFT_TYPE = 1
    double precision, parameter :: PI = 3.141592653589793d0
    double precision, parameter :: DAY = 3600.d0*24.d0
    double precision, parameter :: WEEK = 7d0*DAY
    double precision, parameter :: MONTH = 30*DAY
    double precision, parameter :: YEAR = 365*DAY
    
    ! Variables
    integer :: FAULT_TYPE = 0
    integer :: SOLVER_TYPE = 0
    
end module constants

File: data/code_pairs/cpp/402.cpp
#ifndef CONSTANTS_H
#define CONSTANTS_H

class Constants {
public:
    static constexpr bool BIN_OUTPUT = false;
    static constexpr int FFT_TYPE = 1;
    static constexpr double PI = 3.141592653589793;
    static constexpr double DAY = 3600.0*24.0;
    static constexpr double WEEK = 7.0*DAY;
    static constexpr double MONTH = 30*DAY;
    static constexpr double YEAR = 365*DAY;
    
    // Variables
    static int FAULT_TYPE;
    static int SOLVER_TYPE;
};

// Static member definitions
int Constants::FAULT_TYPE = 0;
int Constants::SOLVER_TYPE = 0;

#endif // CONSTANTS_H